test/providers/curation/processTest.js

```diff
@@ -7,7 +7,8 @@ const memoryQueue = require('../../../providers/queueing/memoryQueue')
 const sinon = require('sinon')
 
 describe('Curation queue processing', () => {
-  it('handles opened message', async () => {
+  it('handles opened message', async function () {
+    this.timeout(12000)
```

Comment:
Can you tell me more about why you added time outs on these tests? I'm sure it's for a good reason, but I would love to understand this better.

Reply:
I added some delay here with comments. https://github.com/clearlydefined/service/pull/878/files#r720403603. This is some experience from the CLA-Assistance.